js/listeners.js

```diff
@@ -26,7 +26,8 @@ function handleClick(a_canvas, e) {
     } else {
         // will fire all the subscribed functions (ex. update UI >>> which use drawNode)
         if (!UI.delete)
```

Comment:
you added a new statement to the if but you add the {}
- this broke the delete logic
### fix:
	`if (!UI.delete) {
		console.log(clickedPoint);
        graph.addNode(new GraphPoint(clickedPoint.x, clickedPoint.y));
	}`